src/map/pc_groups.h

```diff
@@ -57,6 +57,7 @@ enum e_pc_permission {
 	PC_PERM_DISABLE_STORE       = 0x1000000,
 	PC_PERM_DISABLE_EXP         = 0x2000000,
 	PC_PERM_DISABLE_SKILL_USAGE = 0x4000000,
+	PC_PERM_BYPASS_NOSTORAGE	= 0x8000000,
```

Comment:
Tabs should be only at start of line, for everything else, indent should be by spaces

Reply:
Okay =]